composites/SnippetPreview/components/Icon.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+/**
+ * Returns the Icon component.
+ *
+ * @param {object} props Component props.
+ *
+ * @returns {ReactElement} Icon component.
+ */
+export const Icon = ( props ) => {
+	const IconComponent = styled( props.icon )`
+		float: left;
```

Comment:
Not sure it should be floated left by default. Are we assuming just one specific usage?